variables.tf

```diff
@@ -140,6 +140,12 @@ variable "time_limit_ms" {
   type        = number
 }
 
+variable "cors_access_control_max_age" {
+  description = "The amount of time in seconds that the browser should cache the preflight response"
+  default     = 5
```

Comment:
According to [the docs](https://docs.snowplow.io/docs/pipeline-components-and-applications/stream-collector/configure/#:~:text=collector.cors.accessControlMaxAge), the default value is 60 minutes.

Reply:
I'd stick with the upstream default.

Reply:
updated